src/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -0,0 +1,192 @@
+import { Box, chakra, useMultiStyleConfig } from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+
+import ComponentWrapper from "../ComponentWrapper/ComponentWrapper";
+
+/* Define all third party services, will be used for type check
+ * and also used to create the AudioType
+ */
+const thirdPartyServices = ["libsyn", "soundcloud", "spotify"] as const;
+export type ThirdPartyAudioType = typeof thirdPartyServices[number];
+export type AudioType = ThirdPartyAudioType | "file";
+
+export interface AudioPlayerProps {
+  /** Required string used to specify the type of audio playback */
+  audioType: AudioType;
+  /** Optional className you can add in addition to `audio-player` */
+  className?: string;
+  /** Optional string to set the text for the audio player description */
+  descriptionText?: string;
+  /** Optional string to set a code snippet provided by Libsyn, SoundCloud or Spotify; the
+   * `AudioPlayer` component will accept the `embedCode` prop or the `filePath` prop
+   */
+  embedCode?: string;
+  /** Optional string to set the audio file, the path can be relative or absolute
+   * referring to a locally hosted file, or a fully qualified URL pointing to a locally hosted file
+   * or to another domain altogether.
+   * IMPORTANT: This prop will not be used for this version of the AudioPlayer component
```

Comment:
It would be good to clarify what "this version" means. It could be `1.1.0` or `1.1.x` but then we don't know when the `filePath` will be working. For now, maybe saying the inverse that "this prop won't be used until a future version" will be better. Or maybe we should omit this prop to avoid confusion, @bigfishdesign13 ?

Reply:
If the code for this prop has already been added, I would suggest commenting the code and adding a `TODO` note with the code.  It can be vague and say that support for the prop will be added in a future release.